src/backends/policy_engine/souffle/dp_testdata/dp_analysis_1q_failing.ir

```diff
@@ -0,0 +1,14 @@
+//modeling a single query that fails the dp analysis
```

Comment:
In these comments at the top of the files, could you expand a bit more on why these should pass or fail the dp analysis? It is a bit hard to follow just reading the IR language.

Reply:
Done. LMK if you think I should add more detail.